CHANGELOG.md

```diff
@@ -1,6 +1,97 @@
+* The guide to install the Container Service Extension v4.0 now additionally explains how to install the
+  Kubernetes Container Clusters UI Plugin ([#1059](https://github.com/vmware/terraform-provider-vcd/pull/1059))
+* `vcd_external_network_v2` resource and data source support IP Spaces on VCD 10.4.1+ by adding
+  `use_ip_spaces` and `dedicated_org_id` fields ([#1062](https://github.com/vmware/terraform-provider-vcd/pull/1062))
+* `vcd_nsxt_edgegateway` resource supports IP Spaces by not requiring `subnet` specification
+  ([#1062](https://github.com/vmware/terraform-provider-vcd/pull/1062))
+* Resource and data source `vcd_nsxt_alb_virtual_service` support IPv6 on VCD 10.4.0+ via new
+  `ipv6_virtual_ip_address` ([#1071](https://github.com/vmware/terraform-provider-vcd/pull/1071))
+* Resource and data source `vcd_network_routed_v2` support Dual-Stack mode using
+  `dual_stack_enabled` and `secondary_gateway`, `secondary_prefix_length`,
+  `secondary_static_ip_pool` fields ([#1071](https://github.com/vmware/terraform-provider-vcd/pull/1071))
+* Resource and data source `vcd_network_isolated_v2` support Dual-Stack mode using
+  `dual_stack_enabled` and `secondary_gateway`, `secondary_prefix_length`,
+  `secondary_static_ip_pool` fields ([#1071](https://github.com/vmware/terraform-provider-vcd/pull/1071))
+* Resource and data source `vcd_nsxt_network_imported` support Dual-Stack mode using
+  `dual_stack_enabled` and `secondary_gateway`, `secondary_prefix_length`,
+  `secondary_static_ip_pool` fields ([#1071](https://github.com/vmware/terraform-provider-vcd/pull/1071))
+* Resource and data source `vcd_nsxt_network_dhcp_binding` support `dhcp_v6_config` config ([#1071](https://github.com/vmware/terraform-provider-vcd/pull/1071))
+* Validate possibility to perform end to end IPv6 configuration via additional tests ([#1071](https://github.com/vmware/terraform-provider-vcd/pull/1071))
+* Resource `vcd_vdc_group` adds new field `remove_default_firewall_rule` to remove default
+  Distributed Firewall Rule after creation ([#1076](https://github.com/vmware/terraform-provider-vcd/pull/1076))
+* The attribute `description` of `vcd_vm_placement_policy` is now Computed, as latest VCD versions set a default description
+automatically if it is not set ([#1082](https://github.com/vmware/terraform-provider-vcd/pull/1082))
+
+### BUG FIXES
+* Fix `SYSTEM` scope data source `vcd_nsxt_app_port_profile` when multiple NSX-T managers are configured ([#1065](https://github.com/vmware/terraform-provider-vcd/pull/1065))
+* Fix inconsistent `security_profile_customization` field during `vcd_nsxt_ipsec_vpn_tunnel` update ([#1072](https://github.com/vmware/terraform-provider-vcd/pull/1072))
+* Fix Issue #1066 - Not possible to handle more than 128 storage profiles ([#1073](https://github.com/vmware/terraform-provider-vcd/pull/1073))
```

Comment:
Should we add a link to the issue? Not sure if it will be done by GitHub, but at least it is not done in the rendered view.

Reply:
added